frontend/test/src/container/Account.spec.js

```diff
@@ -115,12 +115,12 @@ describe('<Account />', () => {
     })
 
     describe('handleSubmitNameOrEmail', () => {
-      it('updateUserNameCallBack should be called when the function handleSubmitNameOrEmail() is called with a new Name', (done) => {
+      it('updateUserPublicNameCallBack should be called when the function handleSubmitNameOrEmail() is called with a new Name', (done) => {
```

Comment:
take advantage of this PR to rename this test. It should start with "should"

Reply:
ok, I'll rename all the tests in that file to be consistent